### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by lxndrblz** [lxndrblz/anatole#198] [Releases]
# New Features
* Added Japanese Translation
* Allow an option for adjusting sidebar-content ratio
# Fixes
* Fixed styling of ordered and unordered lists
* Fixed a spacing issue for apostrophes and quotation marks on Windows systems
* Fixed a circumstance where the minification of assets would fail, when the theme is deployed through Cloudflare 
* Improved documentation

<hr /><em>This discussion was created from the release <a href='https://github.com/lxndrblz/anatole/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by ashtheish** [comment]
Hello! 

I try to use Anatole as my hugo theme,but I run into some problems,when I follow the [guideline of Anatole's wiki](https://github.com/lxndrblz/anatole/wiki/%F0%9F%92%85-Appearance#adjusting-the-sidebarcontent-ratio) pages,and simply change the params of `contentratio` inside the config file, and refresh my site,then... nothing changes!
(my expected output is sidebar:content = 3:7,so I set `contentratio = 0.7`,same as the example but it doesn't work.)

**Reply by lxndrblz** [reply]
Hi there,

Do you have a both a `params.toml` and `config.toml` file or just a `config.toml`? In the former case you would need to put it into the params file. If you just have a `config.toml` file, you will have to put it under the `[params]` section header, as shown in the documentation.

https://github.com/lxndrblz/anatole/wiki/%F0%9F%92%85-Appearance#adjusting-the-sidebarcontent-ratio

Hope that helps.
